feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/view/stakingTarget/StakingTargetModel.kt

```diff
@@ -0,0 +1,18 @@
+package io.novafoundation.nova.feature_staking_impl.presentation.view.stakingTarget
+
+import androidx.annotation.ColorRes
+
+class StakingTargetModel(
+    val title: String,
+    val subtitle: String?,
+    @ColorRes val subtitleColorRes: Int,
+    val icon: Icon?
+) {
+
+    sealed interface Icon {
+
+        class Quantity(val quantity: String) : Icon
+
+        class Drawable(val drawable: android.graphics.drawable.Drawable) : Icon
```

Comment:
I suggest to store our `Icon` class for better abstraction inside `Drawable` variante (might require renamings of parent `Icon` interface)